.github/workflows/cd.yml

```
@@ -44,73 +30,7 @@ jobs:
-            # Check if web container is healthy
-            echo "Checking web container logs:"
-            docker logs backend_theater --tail 20
-            
-            # Check nginx container logs
-            echo "Checking nginx container logs:"
-            docker logs nginx --tail 20
-            
-            echo "Deployment completed successfully for branch: $BRANCH"
-          EOF
+          ssh ${{ secrets.EC2_USER }}@${{ secrets.EC2_HOST }} \
+                'bash /home/ubuntu/src/online-cinema-project/commands/deploy.sh'
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)